### ISSUE TITLE
Fix header

### ISSUE THREAD

**Issue by mameehan5** [ctowery/tesla_website2019_dev#1]
Need to fix the header of the website so it does not cover content on home, personal pages, etc.

**Comment by ctowery** [comment]
If anyone knows how to limit a CSS stylesheet to overriding the default theme in only a single semantic block of HTML5 (aka header, footer, article, section, aside) that would be amazing information. Academic provides this option for any single Markdown file of content, but I don't know how the Academic theme or Hugo is then achieving this magic.

**Comment by ctowery** [comment]
Issue fixed by latest commit.

**Comment by ctowery** [comment]
Nope. Did not fix the problem for all pages, only the home page. Gonna have to figure this out a bit more.
